projects/core/src/features-config/feature-toggles/config/feature-toggles.ts

```diff
@@ -997,12 +991,11 @@ export const defaultFeatureToggles: Required<FeatureTogglesInterface> = {
   searchBoxV2: false,
   trendingSearches: false,
   pdfInvoicesSortByInvoiceDate: true,
-  storeFrontLibCardParagraphTruncated: true,
   useProductCarouselBatchApi: false,
   productConfiguratorAttributeTypesV2: true,
   propagateErrorsToServer: false,
   ssrStrictErrorHandlingForHttpAndNgrx: false,
-  productConfiguratorDeltaRendering: false,
+  productConfiguratorDeltaRendering: true,
```

Comment:
is this change really related to this PR?

Reply:
Yes, see the updated PR description